### DISCUSSION TITLE
react-datepicker-compat week selection prop

### DISCUSSION THREAD

**Discussion by flytejon** [microsoft/fluentui#28304] [Q&A] [answered]
Hi, 

Is there any plans to add a week selection prop to the `@fluentui/react-datepicker-compat`, similar to the [@fluentui/react-northstar datepicker](https://fluentsite.z22.web.core.windows.net/0.66.2/components/datepicker/definition?showCode=true&showRtl=false&showTransparent=false&showVariables=false#types-week)?

Cheers
Jon

![image](https://github.com/microsoft/fluentui/assets/74728196/8fec8bbd-1dd5-45a5-a8bf-b7f603097ded)

**Comment by sopranopillow** [comment, marked as answer]
Hi @jwccit, this is already possible! here's how to do it: https://codesandbox.io/s/patient-wave-mxpxsv. Will add an example to the docsite next week as well.
